odiglet/pkg/kube/runtime_details/instconfig_controller.go

```diff
@@ -19,12 +19,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-type InstrumentationConfigReconciler struct {
+// replaced by odiglet/pkg/kube/runtime_details/instrumentationconfigs_controller.go
+// which does not rely on the RuntimeDetailsInvalidated flag
+// left here until the migration is complete
+type DeprecatedInstrumentationConfigReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
 }
 
-func (i *InstrumentationConfigReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+func (i *DeprecatedInstrumentationConfigReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
nit:
You can mark this function with a comment of `Deprected:` -  some of the IDEs (like vscode) identify this and mark any call to those functions.